apps/studio/src/features/dashboard/SiteList.tsx

```diff
@@ -115,13 +116,47 @@ const SuspendableSiteList = (): JSX.Element => {
       <Flex flexDirection="column" gap="2rem">
         <SimpleGrid columns={3} gap="2.5rem" width="100%">
           {sites.map((site) => (
-            <Card key={site.id} width="100%">
-              <CardHeader>
-                <Link href={`/sites/${site.id}`} as={NextLink}>
-                  {site.name}
-                </Link>
-              </CardHeader>
-            </Card>
+            <Link
+              href={`/sites/${site.id}`}
+              as={NextLink}
+              textDecoration="none"
+              color="links.neutral-hover"
+              _hover={{
+                textDecoration: "none",
+                color: "links.neutral-hover",
+              }}
+            >
+              <Flex
+                key={site.id}
+                flexDirection="column"
+                gap="1rem"
+                width="100%"
+              >
+                <Image
+                  src={site.config.logoUrl}
+                  alt={site.name}
+                  borderRadius="0.5rem"
+                  border="1.5px solid #EDEDED"
+                  width="100%"
+                  height="100%"
+                  objectFit="cover"
+                  aspectRatio="1/1"
+                  backgroundColor="white"
+                  fallbackSrc="/isomer-sites-placeholder.png"
+                />
+                <Flex flexDirection="column" gap="0.5rem">
+                  <Text
+                    textStyle="h6"
+                    noOfLines={1}
+                    overflow="hidden"
+                    textOverflow="ellipsis"
+                  >
+                    {site.name}
+                  </Text>
+                  {/* <Text textStyle="body-2"></Text> */}
```

Comment:
should we remove this!

Reply:
placeholder because idk how to retrieve the site url or get the "unpublished" state at the moment 🥲

Reply:
ah actually we don't quite have a `site.url` in our db but we have `siteName` (insnide our blob) - we can consider adding it here but that's pretty expensive (compared to other APIs but shud be less than 1kb)

Reply:
okay i will just remove that for now, i don't think it's worth fetching another such API for this page (for now)